src/Admin/ORM/MediaAdmin.php

```diff
@@ -49,16 +49,23 @@ protected function configureDatagridFilters(DatagridMapper $filter): void
             ->add('height')
             ->add('contentType');
 
-        $providers = [];
+        $providersChoices = [];
 
-        $providerNames = $this->pool->getProviderNamesByContext($this->getPersistentParameter('context', $this->pool->getDefaultContext()));
-        foreach ($providerNames as $name) {
-            $providers[$name] = $name;
+        $providers = $this->pool->getProvidersByContext($this->getPersistentParameter('context', $this->pool->getDefaultContext()));
+        foreach ($providers as $provider) {
+            $name = $provider->getName();
+            $translatedName = $this->getTranslator()->trans(
+                $name,
+                [],
+                $provider->getProviderMetadata()->getDomain() ?? 'SonataMediaBundle'
```

Comment:
Maybe this SonataMediaBundle shouldn't be hardcoded here

Reply:
`$this->getTranslationDomain()` ?

Reply:
Yep, changed, it works.